src/request_parser.rs

```diff
@@ -14,3 +42,54 @@ pub trait RequestParser {
     /// Returns `Err(())` if this request does not match any known operation on this API.
     fn parse_operation_id(req: &Request) -> Result<&'static str, ()>;
 }
+
+#[cfg(test)]
+mod context_tests {
+    use super::*;
+    use hyper::{Method, Uri};
+    use std::str::FromStr;
+
+    //This is not needed in hyper > v12, Method::from_bytes can be used
```

Comment:
What's "This"?

Reply:
Ah, something I deleted and left in the accompanying comment